### PR TITLE
faq.js file added

### LINKED ISSUE #118: migrate faq file to new location
**Is your feature request related to a problem? Please describe.**
Yes, the current location to the faq.md might change. also, a user might not want to visit the github repo for the faq.

**Describe the solution you'd like**
* An independent HTML or a file which contains the same faq as the faq.md file in this repo. 
* the idea is to decouple the faq from the repo. 
* do make sure to open a new tab when a visitor clicks on the faq file.
*  update the relevant links in the repo so that the migration is implemented.

**Describe alternatives you've considered**
no.

**Additional context**
agreed by the mods meeting to do this.

### PR DESCRIPTION
**Pull request by shreyakapoor08**
Fixes #118 
FAQ.js file added

### CODE REVIEW

File: FAQ.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import React from "react";
```
Comment (coderwassananmol) [inline FAQ.js:1]: Don't make it a class component. Keep a stateless component for this.
Also, I believe Question and Answers can be two separate components and they can be picked up from constants.js or maybe make a new file called strings.js and read from there to keep things tidy.

File: FAQ.md [hunk 2]
```diff
@@ -1,25 +0,0 @@
-<b>Frequently Asked Questions</b>
```
Comment (coderwassananmol) [inline FAQ.md:1]: Why deleted this file?

File: FAQ.js (hunk 1 above)
Comment (coderwassananmol) [reply FAQ.js:1]: Also, place it under pages directory. It will directly get served from there.

File: components/Header.js [hunk 3]
```diff
@@ -1,91 +1,154 @@
-import React, { Component } from 'react'
```
Comment (coderwassananmol) [inline components/Header.js:1]: Why so many changes are made to this file? Just the link has to be added for faq.
Let me know if you have any questions.

**Comment by rajat2502** [comment]
@shreyakapoor08 Any progress on this?

**Comment by shreyakapoor08** [comment]
I was not able to understand it 😅 I messaged Anmol on slack. I am not able to understand how to do it

**Comment by coderwassananmol** [comment]
> I was not able to understand it 😅 I messaged Anmol on slack. I am not able to understand how to do it

Let me know on slack/here if you need some help.

**Comment by coderwassananmol** [comment]
Duplicate of #132

**Pull request closed without merging**